### DISCUSSION TITLE
Support Dynamic packet pacing please

### DISCUSSION THREAD

**Discussion by PlayXboxtion963** [LizardByte/community#501] [Sunshine]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the latest beta/pre-release?

This issue is present in the latest pre-release

### Describe the Bug

Such like moonlight author referred. Only you fix it moonlight can get more than 1Gbps video stream. Please
[https://github.com/moonlight-stream/moonlight-qt/commit/f3a75e8e762a08f8498ee212745fbef0d82b0bfd#commitcomment-145813987](url)

### Host Operating System

Windows

### Operating System Version

11

### Architecture

64 bit

### Sunshine commit or version

latest

### Package

Windows - portable

### GPU Type

AMD

### GPU Model

rx580

### GPU Driver/Mesa Version

3

### Capture Method

Desktop Duplication API (Windows)

### Config

```Shell
null
```

### Relevant log output

```shell
null
```

**Comment by devMorpheus** [comment]
@LizardByte what you think about it?
